src/test/java/picard/cmdline/PicardCommandLineTest.java

```diff
@@ -59,4 +58,10 @@ public void testLaunchAllCommandLineProgramsWithBarclayParser() {
                 }
         );
     }
+
+
+    @Test
+    public void testPrintUsage() {
+        Assert.assertEquals(new PicardCommandLine().instanceMain(new String[]{"-h"}), 1);
+    }
 }
```

Comment:
need final nl to remove annoying red mark in github.

Reply:
Hello @yfarjoun , all requested changes are done